joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
+The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that can reduce the memory footprint of phylogenetic information.
 
 # Statement of Need
 
 <!-- was going to say a rich history of sophisticated phylogenetic analyses... -->
 *In silico* evolution work enjoys a rich history of phylogenetic record-keeping and analysis, and many systems facilitate tracking phylogenies [@ray1992evolution;@ofria2004avida;@bohm2017mabe;@de2012deap;@Garwood_REvoSim_Organism-level_simulation_2019].
 However, to our knowledge, no other general-purpose perfect phylogeny tracking library exists; prior work has used bespoke system- or framework-specific implementations.
-In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any system containing replicating entities.
+In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any simulation of replicating entities.
 
-Two other general-purpose libraries for recording phylogenetic information do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
-However, they allow different modes of phylogenetic tracking than Phylotrack does.
+Two other general-purpose libraries for phylogenetic tracking do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
+However, they pursue alternate modes of record keeping to Phylotrack.
 Whereas Phylotrack uses a graph-based approach to perfectly record asexual phylogenies, the hstrat library implements hereditary stratigraphy, a recently developed method that allows robust decentralized phylogenetic tracking in parallel and distributed systems at the cost of a tunable reduction in accuracy [@moreno2022hstrat].
 APOGeT, in turn, focuses on tracking speciation in sexually-reproducing populations [@godin2019apoget].
 
 Vast amounts of bioinformatics-oriented phylogenetics software is also available.
 These programs' purposes typically include
-- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic]
-- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating]
-- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic]
-- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython]
+- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic],
+- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating],
+- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic], and
+- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython].
```

Comment:
Okay, yeah, I read up on it and it looks like you're right about the punctuation here!

Reply:
used to be a model un kid which makes me _very_ familiar with sentence clauses in giant lists (resolutions are formatted as one giant list with a single period at the end OwO)